code/aniplanrr.py

```diff
@@ -4,48 +4,72 @@
 from sonarr import *
 
 # if logging is true
-if LOGGING is not None:
+if LOGGING:
     pr("Logging is enabled")
 else:
     pr("Logging is disabled")
 
 ignoreList = loadIgnoreList()
 mapping = loadMappingList()
 
+def runSonarr(sonarr, aniList):
+    if LOGGING:
+        pr("Getting Sonarr List")
+    sonarrList = getSonarrList(sonarr)
+    if sonarrList is None:
+        pr("Sonarr List is empty")
+        #stop execution
+        return False
+    # Remove obvious matches
+    newShowList = diffDicts(aniList, sonarrList)
+    # Remove less obvious matches via IDs/Mapping
+    
+    newShowList = indexSonarrList(sonarr, newShowList, mapping, sonarrList)
+    if LOGGING:
+        pr("Found " + str(len(newShowList)) + " new shows to add to Sonarr")
+    
+    # send each item in newShows to get_id_from_sonarr
+    sendToSonarr(sonarr, newShowList, sonarrList)
+
+def runRadarr(radarr, aniMovieList):
+    if LOGGING:
+        pr("Getting Radarr List")
+    radarrList = getRadarrList(radarr)
+    if radarrList is None:
+        pr("Radarr List is empty")
+        return False
+    newMoviesList = diffDicts(aniMovieList, radarrList)
+    newMoviesList = indexRadarrList(radarr, newMoviesList, mapping, radarrList)
+    if LOGGING:
+        pr("Found " + str(len(newMoviesList)) + " new movies to add to Radarr")
+    sendToRadarr(radarr, newMoviesList, radarrList)
 
 def main():
     if LOGGING:
         pr("Getting AniList for " + ANILIST_USERNAME)
     [aniList, aniMovieList] = getAniList(str(ANILIST_USERNAME))
     # filter anilist if anilist[2] is in ignorelist
-    aniList = [x for x in aniList if x['anilistId'] not in ignoreList]
-    aniMovieList = [
-        x for x in aniMovieList if x['anilistId'] not in ignoreList]
+    if not (RETRY):
```

Comment:
Unless I'm mistaken, this would make it so the AniList isn't filtered if you don't have RETRY=True. The idea was to always filter the list, but to only populate with failures if if Retry=False

Reply:
It would make it so that the AniList isn't filtered if you DO have RETRY=True. Lot of negatives going on here.
This way, if you want to debug something, you can have it try everything.
Added a commit for a 'manual' option that restores previous functionality, while allowing new functionality, if this is okay with you.